### DISCUSSION TITLE
Data source Azure Blob

### DISCUSSION THREAD

**Discussion by asztymelski** [grafana/grafana-infinity-datasource#946] [Q&A]
Hi, Im trying to grab data from Azure Storage Account, and once im adding Authentication method all is fine:
![image](https://github.com/user-attachments/assets/31caa91a-e428-4da4-b1a1-fe7bda21dfd5)
but when Im trying retrieve the data Im receiving 404 BlobNotFound
![image](https://github.com/user-attachments/assets/28313919-614b-47b6-907f-dbce122fec60)

im doing something wrong or that kid of authentication doesnt work?

**Comment by yesoreyeram** [comment]
Looks like the blob name provided is wrong. Also are you latest version of the plugin?

**Reply by asztymelski** [reply]
Thanks for reply yesoreyeram, 
yes it is
![image](https://github.com/user-attachments/assets/849013ac-02c6-4dd7-8699-9bfaddc16b62)
and my container name is insights-metrics-pt1m
![image](https://github.com/user-attachments/assets/a7783080-3592-40ce-81d2-34d0a2061121)
and the blob is resourceId=
![image](https://github.com/user-attachments/assets/38824c08-7676-4869-a2c9-b49acba241d3)

**Comment by asztymelski** [comment]
I see. The solution is actually quite straightforward. My goal was to visualize Azure metrics that I'm storing in my Storage Account. However, I've discovered that the metrics exported by Azure are not in valid JSON format, which makes them unusable as is.
I'm wondering if there's a convenient method to visualize Azure metrics in Grafana. Or is the only option to use an exporter that converts the data to Prometheus-style metrics?
